sklearn/neural_network/_multilayer_perceptron.py

```diff
@@ -189,12 +190,16 @@ def _forward_pass_fast(self, X):
         X : {array-like, sparse matrix} of shape (n_samples, n_features)
             The input data.
 
+        validation : boolean, default=True
```

Comment:
```suggestion
        validation : bool, default=True
```

For reference, this is item 1 in our [documentation style guide](https://scikit-learn.org/dev/developers/contributing.html#guidelines-for-writing-documentation)

Reply:
I didn't realise this existed (and wasn't out of date)!